solutions/observability/get-started/quickstart-monitor-hosts-with-opentelemetry.md

```diff
@@ -9,138 +9,4 @@ applies_to:
-
-Logs are collected from setup onward, so you won’t see logs that occurred before starting the EDOT Collector. The default log path is `/var/log/*`. To update the path, modify `otel.yml`.
-
-::::
-
-:::::
-
-
-Under **Visualize your data**, you’ll see links to **Logs Explorer** to view your logs and **Hosts** to view your host metrics.
-
-
-## Gain deeper insight into your host data  [_get_value_out_of_your_data]
-
-After using the Hosts page and Discover to confirm you’ve ingested all the host logs and metrics you want to monitor, use Elastic {{observability}} to gain deeper insight into your host data with the following capabilities and features:
-
-* In the [Infrastructure UI](../../../solutions/observability/infra-and-hosts/analyze-infrastructure-host-metrics.md), analyze and compare data collected from your hosts. You can also:
-
-    * [Detect anomalies](../../../solutions/observability/infra-and-hosts/detect-metric-anomalies.md) for memory usage and network traffic on hosts.
-    * [Create alerts](../../../solutions/observability/incident-management/create-manage-rules.md) that notify you when an anomaly is detected or a metric exceeds a given value.
-
-* In the [Logs Explorer](../../../solutions/observability/logs/logs-explorer.md), search and filter your log data, get information about the structure of log fields, and display your findings in a visualization. You can also:
-
-    * [Monitor log data set quality](../../../solutions/observability/data-set-quality-monitoring.md) to find degraded documents.
-    * [Run a pattern analysis](../../../explore-analyze/machine-learning/machine-learning-in-kibana/xpack-ml-aiops.md#log-pattern-analysis) to find patterns in unstructured log messages.
-    * [Create alerts](../../../solutions/observability/incident-management/create-manage-rules.md) that notify you when an Observability data type reaches or exceeds a given value.
-
-* Use [machine learning](../../../explore-analyze/machine-learning/machine-learning-in-kibana.md) to apply predictive analytics to your data:
-
-    * [Detect anomalies](../../../explore-analyze/machine-learning/anomaly-detection.md) by comparing real-time and historical data from different sources to look for unusual, problematic patterns.
-    * [Analyze log spikes and drops](../../../explore-analyze/machine-learning/machine-learning-in-kibana/xpack-ml-aiops.md#log-rate-analysis).
-    * [Detect change points](../../../explore-analyze/machine-learning/machine-learning-in-kibana/xpack-ml-aiops.md#change-point-detection) in your time series data.
-
-
-Refer to the [Elastic Observability](../../../solutions/observability.md) for a description of other useful features.
\ No newline at end of file
+For information on monitoring your hosts with OpenTelemetry, refer to the [Elastic OpenTelemetry documentation](https://elastic.github.io/opentelemetry/use-cases/host-metrics/).
```

Comment:
Maybe we should delete this page altogether and add a link to https://elastic.github.io/opentelemetry/use-cases/host-metrics/ directly to https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/985/solutions/observability/get-started#quickstarts-overview. What do you think?

Reply:
👍 That makes more sense